ably/rest/auth.py

```diff
@@ -82,7 +82,10 @@ async def get_auth_transport_param(self):
             return {"key": f"{key_name}:{key_secret}"}
         elif self.__auth_mechanism == Auth.Method.TOKEN:
             token_details = await self._ensure_valid_auth_credentials()
-            return {"accessToken": token_details.token}
+            auth_credentials = {"accessToken": token_details.token}
+            if token_details.client_id:
```

Comment:
As per [`RTN2d`](https://sdk.ably.com/builds/ably/specification/main/features/#RTN2d), this should be set to the client_id from Options rather than the client_id on the token.